src/main/java/it/pagopa/pn/deliverypush/legalfacts/DocumentComposition.java

```diff
@@ -39,6 +39,8 @@ public enum TemplateType {
         NOTIFICATION_VIEWED("documents_composition_templates/NotificationViewedLegalFact.html"),
         AAR_NOTIFICATION("documents_composition_templates/NotificationAAR.html"),
         AAR_NOTIFICATION_RADD("documents_composition_templates/NotificationAAR_RADD.html"),
+        AAR_NOTIFICATION_RADD_ALT("documents_composition_templates/NotificationAAR_RADDalt.html")
+        AAR_TEST("documents_composition_templates/NotificationAAR_RADDalt.html"),
```

Comment:
@SarahDonvito why adding this other variable only for testing?

Reply:
I think it was the variable added to test during development. I'm going to remove it